display/characters.py

```diff
@@ -41,21 +45,32 @@ def moveX(self, amount):
     def moveY(self, amount):
         self.rect.y += amount
 
+    def makeCollisionBoxHaveRightWidthAndHeightAndXCoordinateAndYCoordinate(self):
+        print(CHARSTATES)
+        currentChar_state = CHARSTATES['chars'][self.char_name]['states'][self.currentStage]
+
+        self.collision_rect.y = self.rect.y + (480 - currentChar_state['height']*10)
+        self.collision_rect.x = self.rect.x + (160 - currentChar_state['width']*5)
+        self.collision_rect.width = currentChar_state['width']
+        self.collision_rect.height = currentChar_state['height']
 
 class ZUCC(Character):
 
     def __init__(self):
 
-        self.currentStage = 0
         self.stages = os.listdir("assets/zucc/")
         self.stages.sort()
 
         super().__init__()
 
+        self.char_name = "zucc"
```

Comment:
please use `__name__`

Reply:
please do the needful and use _`_g`o_aw`ay_`_